### ISSUE TITLE
Updating to version 1.6.0

### ISSUE THREAD

**Issue by jimbrew** [Excel-DNA/ExcelDna#552]
Hello I am trying to update the nuget package on a .net framework 4.8 excel add in.

The current version I am using is

- ExcelDna.AddIn v1.1.1
- ExcelDna.Integration v1.1.0

I want to update these both to v1.6. However, I get the error messages -

```
20>  ExcelDnaPack: bin\x86\Debug\FloXL2020-AddIn.dna -> bin\x86\Debug\FloXL2020-AddIn-packed.xll
20>MSBUILD : PackExcelAddIn error DNA1899655818:   ~~> ERROR: Reference with Path: Microsoft.VisualStudio.Tools.Applications.Runtime.dll and Name:  NOT FOUND.
20>MSBUILD : PackExcelAddIn error DNA1899655818: System.InvalidOperationException:   ~~> ERROR: Reference with Path: Microsoft.VisualStudio.Tools.Applications.Runtime.dll and Name:  NOT FOUND.
20>MSBUILD : PackExcelAddIn error DNA1899655818:    at ExcelDna.PackedResources.ExcelDnaPack.PackDnaLibrary(Byte[] dnaContent, String dnaDirectory, ResourceUpdater ru, Boolean compress, Boolean multithreading, List`1 filesToPublish)
20>MSBUILD : PackExcelAddIn error DNA1899655818:    at ExcelDna.PackedResources.ExcelDnaPack.Pack(String dnaPath, String xllOutputPathParam, Boolean compress, Boolean multithreading, Boolean overwrite, String usageInfo, List`1 filesToPublish)
20>MSBUILD : PackExcelAddIn error DNA1899655818:    at ExcelDna.AddIn.Tasks.PackExcelAddIn.Execute()
```

**Comment by acher1965** [comment]
My 0.02$: I got this yesterday, and I thought it was because yesterday WindowsDefender started flagging 1.6.0 as malware (wrongly etc etc), and the Remove action deleted the nuget 1.6.0 package.
I worked around it by by following a colleague's suggestion:
close VisualStudio, delete everything inside ./nuget/packages, then reopenVS: it took a little while to get all of the nugets again, but this time it did, then I cleaned solution and rebuilt, OK.
So I can't be sure whether windows defender was responsible to begin with or not.
And don't follow my steps... they might make the situation worse on your machine :-)
